### ISSUE TITLE
Generating image from API

### ISSUE THREAD

**Issue by melohagan** [Budibase/budibase#9461]
### Discussed in https://github.com/Budibase/budibase/discussions/9454

<div type='discussions-op-text'>

<sup>Originally posted by **kerbersystems** January 26, 2023</sup>
Hey everyone. So, I'm trying to find out if it's possible to show the PNG result from an API call. Zebra has a free barcode generator API that returns a PNG. I can't seem to get it to display in Budibase as it can't accept that response.</div>

**Comment by bjornclauw** [comment]
Hi @melohagan,
tried something similar a while back (without succes)
I used HTTP GET and used following code inside the transformer:
`retval=Buffer.from(data,'binary').toString('base64');
return retval;`

I tried showing the BASE64 image inside an embed component with no succes.
There were a few characters corrupted/modified in my BASE64 image, i suspect this had something to do with how the schema handles the BASE64 data. (used TEXT for the valuetype of my key)

Ultimately abandoned this idea because I wanted to show the images without needing the server/backend.
